### PR TITLE
Added BSDF construction supports in energy simulation

### PR DESCRIPTION
**Pull request by MingboPeng**
1) import BSDF idf directly in "addToEPLibrary"
2) import multi-EPobjects at a time

### CODE REVIEW

File: src/Honeybee_Add to EnergyPlus Library.py [hunk 1]
```diff
@@ -50,7 +50,32 @@
 
 import scriptcontext as sc
 import Grasshopper.Kernel as gh
+import os
 
+def prepEPObjects(rawStrings):
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Add to EnergyPlus Library.py]: Hi @MingboPeng! This method can fail if the EnergyPlus file is not clean. Instead use Honeybee's `importEPLibraryFromFile` method.

Comment (MingboPeng) [inline src/Honeybee_Add to EnergyPlus Library.py]: @mostaphaRoudsari , where is importEPLibraryFromFile? in Honeybee_Honeybee? I couldn't find it. What do you mean not clean,  like idf file with "!" notes? Does this method returns EPObjects?

Comment (mostaphaRoudsari) [inline src/Honeybee_Add to EnergyPlus Library.py]: Ah! I forgot to copy the link. [Here it is!](https://github.com/mostaphaRoudsari/honeybee/blob/master/src/Honeybee_Honeybee.py#L568-L581)
Yes. The idf file. I have seen several cases that people do stuff that is fine with EP but will have issues for us to import. This method is trying to address that.

Comment (MingboPeng) [inline src/Honeybee_Add to EnergyPlus Library.py]: @mostaphaRoudsari re.findall(), this is so useful.

**Comment by mostaphaRoudsari** [comment]
@MingboPeng any chance that you can send a new PR for this before getting so many conflicts?

@chriswmackey do you have any comments on the overall approach for this integration?

**Comment by MingboPeng** [comment]
Couldn't be sure results make sense, will double check this later, and submit a new pull request.

**Comment by chriswmackey** [comment]
Ah, I'm sorry that I dropped the ball on commenting here.

My only addition to Mostapha's comments would be that I don't want to add functionality to the "Run Simulation" component without adding it to the OpenStudio component.  So the BSDF code should also be written into this class of the OpenStudio component:
https://github.com/mostaphaRoudsari/honeybee/blob/master/src/Honeybee_Export%20To%20OpenStudio.py#L3358-L3540

**Pull request closed without merging**